reactioncommerce/reaction
# Admin should be able to invite a user with the same name

**Resolves**
#1983 

**Test**
- Login as admin
- Go to the accounts on action menu
- Invite a user by adding the person's name and email address
- If successful, invite another user with the same name but different email address
- You should be able to see the user below where other users are displayed.
